scriptmodules/supplementary/sdl2.sh

```diff
@@ -11,7 +11,7 @@
 
 rp_module_id="sdl2"
 rp_module_desc="SDL (Simple DirectMedia Layer) v2.x"
-rp_module_licence="ZLIB https://hg.libsdl.org/SDL/raw-file/f426dbef4aa0/COPYING.txt"
+rp_module_licence="ZLIB https://raw.githubusercontent.com/libsdl-org/SDL/main/COPYING.txt"
```

Comment:
This was changed to https://raw.githubusercontent.com/libsdl-org/SDL/main/LICENSE.txt (to accommodate Github's automatic license detection).